src/user/store/authorization/actions.ts

```diff
@@ -1,11 +1,17 @@
 import { ActionTree } from 'vuex';
-import { AuthorizationStateInterface, AuthActionEnum, AuthMutationEnum, Actions } from '@/store/authorization/types';
+import {
+  AuthorizationStateInterface,
+  AuthActionEnum,
+  AuthMutationEnum,
+  Actions,
+} from '@/user/store/authorization/types';
 import { RootStateInterface, StoreModuleEnum } from '@/store/types';
+// <<<<<<< HEAD:src/user/store/authorization/actions.ts
```

Comment:
This line is left after the merge, please delete it if you don't need it.

Reply:
done